queries.yaml

```diff
@@ -542,6 +552,23 @@ post_process:
       source_layer: roads
       properties: [layer]
 
+  # supersede existing osm buildings with data from landmark geojson layer
+  - fn: vectordatasource.transform.overlap
+    params:
+      base_layer: buildings
+      cutting_layer: landmarks
+      attribute: superseded
+      target_attribute: superseded
```

Comment:
In all the places...

`superseded_by` instead of `superseded`.

Please use the same family of property names as WOF in Mapzen tradition: 

https://github.com/whosonfirst/whosonfirst-properties/blob/main/properties/wof/superseded_by.json

Reply:
ah wow i had no idea this existed, i'll swap it out